src/mlpack/tests/cv_test.cpp

```diff
@@ -172,4 +187,110 @@ BOOST_AUTO_TEST_CASE(TakesNumClassesTest)
       "Value should be false");
 }
 
+BOOST_AUTO_TEST_CASE(SimpleCVAccuracyTest)
+{
+  // Using the first half of data for training and the rest for validation.
+  // The validation labels are 75% correct.
+  arma::mat data =
+    arma::mat("1 0; 2 0; 1 1; 2 1; 1 0; 2 0; 1 1; 2 1").t();
+  arma::Row<size_t> labels("0 0 1 1 0 1 1 1");
+
+  SimpleCV<LogisticRegression<>, Accuracy> cv(0.5, data, labels);
+
+  BOOST_REQUIRE_CLOSE(cv.Evaluate(), 0.75, 1e-5);
+}
+
+BOOST_AUTO_TEST_CASE(SimpleCVMSETest)
+{
+  // Using the first two points for training and remaining three for validation.
+  // See the test MSETest for more explanation.
+  arma::mat data("0 1 2 3 4");
+  arma::rowvec responses("-1 0 1 3 5");
+
+  double expectedMSE = (0 * 0 + 1 * 1 + 2 * 2) / 3.0;
+
+  SimpleCV<LinearRegression, MSE> cv(0.6, data, responses);
+
+  BOOST_REQUIRE_CLOSE(cv.Evaluate(), expectedMSE, 1e-5);
```

Comment:
This is related to the earlier comment about whether or not to take the `MatType`/`WeightsType`/`PredictionsType` parameters at class construction time, but I think this interface would be nicer to use if the user could basically just do

```
SimpleCV<LinearRegression, MSE> cv(0.6);
cv.Evaluate(data, responses);
```

since that way they could evaluate cross-validation on multiple different datasets (if for some reason they wanted to do that).  What do you think?